diff --git a/src/main.cpp b/src/main.cpp
index 7b06f1b..2d3c48d 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -24,7 +24,7 @@ APICALL EXPORT PLUGIN_DESCRIPTION_INFO PLUGIN_INIT(HANDLE handle) {
 	}
 #endif
 
-	selection_hook::init();
+	// selection_hook::init();
 
 #define CONF(NAME, TYPE, VALUE)                                                                    \
 	HyprlandAPI::addConfigValue(                                                                     \
